resources/styles/components/exercises.less

```diff
@@ -39,11 +55,42 @@
   }
 
   .page-navigation {
-    .page-navigation-arrows(800px);
+    // .page-navigation-arrows(800px);
 
-    height: @exercise-details-body-height;
     position: absolute;
-    top: 0;
+    top: 20%;
+    z-index: 1;
+    cursor: pointer;
+
+    &.prev {
+      left: 0;
+    }
+
+    &.next {
+      right: 0;
+    }
+
+    .pinned-on & {
+      top: 40%;
+      position: fixed;
+
+      &.prev {
+        left: 40px;
+      }
+
+      &.next {
+        right: 40px;
+      }
+    }
+
+    i.fa {
+      color: #e5e5e5;
```

Comment:
are we using the `Icon` component ?  If so it'll have class `tutor-icon` which might be better to target